test/FhirLoader.Tests.E2E/FHIRLoaderE2ETest.cs

```diff
@@ -93,5 +94,152 @@ public async void BlobFhir_ndjson_Test()
+        /// Throw an exception if the file does not exist.
+        /// </summary>
+        [Fact]
+        public async void LocalFhir_packagewithoutpackagejson_Test()
+        {
+            CommandOptions commandOptions = new()
+            {
+                FhirUrl = _config.FhirURL,
+                PackagePath = @"../../../TestData/packagewithoutpackagejson",
+                BatchSize = _config.Batchsize,//Must be betweeen 1 & 500,Default 500
+                Concurrency = _config.Concurrency//Must be betweeen 1 & 50,Default 50
+            };
+            int response = await Program.Run(commandOptions);
+            Action throwingAction = () =>
+            {
+                throw new ArgumentException();
+            };
+            Assert.Throws<ArgumentException>(throwingAction);
+
+
+        }
+
+        /// <summary>
+        /// Test Case for wrong package folder.
+        /// By passing a wrong path or other nested folder inside the package folder (e.g example, openapi),
+        /// it will check if.index and package json exist, otherwise throw an exception.
+        /// </summary>
+        [Fact]
+        public async void LocalFhir_ifpassingwrongpath_Test()
+        {
+            CommandOptions commandOptions = new()
+            {
+                FhirUrl = _config.FhirURL,
+                PackagePath = @"../../../TestData/package/example",
+                BatchSize = _config.Batchsize,//Must be betweeen 1 & 500,Default 500
+                Concurrency = _config.Concurrency//Must be betweeen 1 & 50,Default 50
+            };
+            int response = await Program.Run(commandOptions);
+            Action throwingAction = () =>
+            {
+                throw new ArgumentException();
+            };
+            Assert.Throws<ArgumentException>(throwingAction);
+
+
+        }
+
+        /// <summary>
+        /// Test Case for Package Files.
+        /// By passing Package Path,it will read all the files inside the package folder and process them to the Fhir URL.
+        /// </summary>
+        [Fact]
+        public async void LocalFhir_package_Test()
+        {
+            CommandOptions commandOptions = new()
+            {
+                FhirUrl = _config.FhirURL,
+                PackagePath = @"../../../TestData/package",
+                BatchSize = _config.Batchsize,//Must be betweeen 1 & 500,Default 500
+                Concurrency = _config.Concurrency//Must be betweeen 1 & 50,Default 50
+            };
+            int response = await Program.Run(commandOptions);
+            Assert.Equal(0, response);
+        }
+
+        /// <summary>
+        /// Check that the json file has the resource type bundle.
+        /// If a file without a bundle resource type exists in a bundle folder, then the tool will skip those files.
+        /// </summary>
+        [Fact]
+        public void CheckIfFileswithoutbundleresourceTypeExist_Test()
```

Comment:
What is this testing exactly? I don't see any execution of the program code in this test.

Reply:
Tool will skip those files which do not have bundle resource type.